share/wake/lib/system/runner.wake

```diff
@@ -55,97 +56,70 @@ export tuple RunnerOutput =
 # A Runner describes a way to invoke a Plan to get a Job
 export tuple Runner =
     export Name: String
-    export Score: Plan => Result Double String
     Fn: Job => Result RunnerInput Error => Result RunnerOutput Error
 
-# Create new Runner given pre- and post-hooks around an existing Runner
-# param name: String
-# param score: type Plan → Result Double String
-#   Called by runJob to produce a score representing the priority of a runner with respect to the given Plan.
-# param pre: type Result RunnerInput Error → Pair (Result RunnerInput Error) a
-#   Called before the job is run, allowing the runner to modify the input to provide the requested resources.
-# param post: type Pair (Result RunnerOutput Error) a → Result RunnerOutput Error
-#   Similar to the pre function but called after the job has run. post is for editing the reported outputs/inputs/usage of the job.
-# param (Runner _ _ run): base runner that the current runner is built on top of
-#   i.e. JSONRunner is built on localRunner.
-
-export def makeRunner name score pre post (Runner _ _ run) =
-    def doit job preInput = match (pre preInput)
-        Pair runInput state ->
-            def runOutput = run job runInput
-            def final _ = post (Pair runOutput state)
+# makeRunner: Hides some of the boiler plate required to create a runner
+#
+# This function requires very advanced wake expierence and should be used with the greatest amount of caution.
+# Callers must ensure at the very least than run calls primJobLaunch and one of the many job "wait" functions.
+# Runners should not wrap other runners instead they should precisely do what they need to complete their task.
```

Comment:
I'm not sure I agree with this line; it's certainly a summation of how this architecture differs from the old one, but at this point we shouldn't have to talk about runners wrapping others going forward, and someone coming at this without historical context might think we're forbidding sharing code between runners or something (that's not just strawmanning the reading, it's the only way my mind makes sense of what it's saying once I "forget" the history).

Reply:
Yeah fair. I guess its supposed to be more of a "word of caution"/"don't make the same mistake we did" type thing

Reply:
It definitely is a great word of caution to have, and after the weekend away I do see good use to keeping something like it in, I just think that we might want to rephrase it a bit to emphasize that it's a historical/advisory note rather than something which describes what is now the new natural inclination.